### PR TITLE
Document pathoplexus_app Helm chart

### PR DESCRIPTION
**Pull request by theosanderson**
It's kind of hard to describe this but I tried to

### CODE REVIEW

File: pathoplexus_app/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Pathoplexus_app
+
+This directory is a tiny Helm chart used to deploy instances of Pathoplexus. It creates a kubernetes resource for an ArgoCD `Application`. _That application_ deploys the (large) _Loculus_ Helm chart. The version (SHA) of Loculus to be used is defined by the `Values.yaml` in this directory.
```
Comment (chaoran-chen) [inline pathoplexus_app/README.md:3]:
```suggestion
This directory is a tiny Helm chart used to deploy instances of Pathoplexus. It creates a kubernetes resource for an ArgoCD `Application`. That application deploys the (large) [Loculus Helm chart](https://github.com/loculus-project/loculus/tree/main/kubernetes). The version of Loculus, that should be used, is defined by the `Values.yaml` in this directory by setting the commit SHA.
```

File: pathoplexus_app/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+All the specific Loculus configuration is defined in `loculus_values` in the parent directory. The end result is that each version (commit) of this _pathoplexus_ repository contains a reference, in this `values.yaml` to a particular version of Loculus, which is used:
+  - to combine with the `monorepo` directory to build the Pathoplexus website image
+  - to specify the version of the Loculus Helm chart to deploy
+  - to specify what versions of the Loculus component to deploy
+
+(Each commit also contains the `loculus_values` file which specifies the specific configuration of that loculus chart, and the `monorepo` directory which specifies what pathopelexus-specific additions to make to the website)
+
+The chart contains its own hardcoded `loculusVersion` value. It also expects to be provided with the following values:
+- `pathoplexusVersion`: this should be the same as the `pathoplexus` SHA of this actual version of the `pathoplexus_app` Helm chart
```
Comment (chaoran-chen) [inline pathoplexus_app/README.md:13]:
```suggestion
- `pathoplexusVersion`: this should be the same as the `pathoplexus` commit SHA of this actual version of the `pathoplexus_app` Helm chart
```

File: pathoplexus_app/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Pathoplexus_app
+
+This directory is a tiny Helm chart used to deploy instances of Pathoplexus. It creates a kubernetes resource for an ArgoCD `Application`. _That application_ deploys the (large) _Loculus_ Helm chart. The version (SHA) of Loculus to be used is defined by the `Values.yaml` in this directory.
+
+All the specific Loculus configuration is defined in `loculus_values` in the parent directory. The end result is that each version (commit) of this _pathoplexus_ repository contains a reference, in this `values.yaml` to a particular version of Loculus, which is used:
```
Comment (chaoran-chen) [inline pathoplexus_app/README.md:5]:
```suggestion
All the specific Loculus configuration is defined in `loculus_values` in the parent directory. The end result is that each version (commit) of this _pathoplexus_ repository contains a reference to a particular version of Loculus in the `values.yaml`, which is used:
```

File: pathoplexus_app/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  - to combine with the `monorepo` directory to build the Pathoplexus website image
```
Comment (chaoran-chen) [inline pathoplexus_app/README.md:6]:
```suggestion
  - to extend the Loculus website with Pathoplexus-specific pages (this is stored in the `monorepo` directory)
```

File: pathoplexus_app/README.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  - to specify the version of the Loculus Helm chart to deploy
+  - to specify what versions of the Loculus component to deploy
```
Comment (chaoran-chen) [inline pathoplexus_app/README.md:8]: What's the difference between them?

**Review by chaoran-chen — APPROVED** [review]
Thank you, this is a good start! I added a few suggestions, feel free to take whever you think is useful.

File: pathoplexus_app/README.md (hunk 5 above)
Comment (theosanderson) [reply pathoplexus_app/README.md:8]: This was a typo and I should have written "components", but actually a better word is "images" which I'll change this to, thanks

**Pull request merged**